### ISSUE TITLE
Add support for <BLANKLINE> keyword

### ISSUE THREAD

**Issue by skycaptain** [catch22/octave-doctest#239]
Python doctest is using the keyword `<BLANKLINE>` for declaring empty lines within a doctest result. Using such a keyword helps the doctest parser finding connected blocks of tests.

See
-    https://docs.python.org/3.7/library/doctest.html#how-are-docstring-examples-recognized
-    https://github.com/catch22/octave-doctest/pull/236#issuecomment-537386844

**Comment by cbm755** [comment]
This would only matter with `-NORMALIZE_WHITESPACE`---by default, we match any whitespace.

But I don't see any problem with some minimal support for these, e.g., replacing them with empty strings in `want` before comparing to `got`.

**Comment by skycaptain** [comment]
Right, I thought of this to be a helper for the regex searching and seperating the doctests. In the end `<BLANKLINE>` gets replaced by a newline character.

**Comment by catch22** [comment]
I don't think we've needed this one so far (and we default to normalizing whitespace since there are differences between Octave and Matlab IIRC). Is this solving any problem that arises when using Sphinx?

**Comment by skycaptain** [comment]
Right. Rethought this. Matlab prints out lots of blank lines, so legability would definitly suffer from such a keyword. But, without having looked at the code; when do you decide what line is still part of the doctest output and what part is of the helptext. Like in this test:
https://github.com/catch22/octave-doctest/blob/master/test/test_whitespace.m
Is it the two empty lines?

**Comment by catch22** [comment]
That's right.
